charts/spire/README.md

```diff
@@ -32,6 +32,16 @@ For production installs, please see [the production example](https://github.com/
 
 ## Upgrade notes
 
+### 0.15.X
+
+The spire-crds chart has been updated. Please ensure you have upgraded spire-crds before upgrading the spire chart.
+
+The chart now includes support for multiple parallel installs of the spire-controller-manager. To facilitate this change, all custom resources now take
+a className property. Out of the box the chart now uses and requires it to be set. If you have not loaded any yourself, the upgrade process will be transparent.
```

Comment:
> If you have not loaded any yourself

What is `any` here?

> uses and requires it to be set

Assuming the className has to be set? Can't we make it such it has a default for the simple spire-system way of installing all in one namespace, so you only have to override for the advanced split namespace install?

> className

What does a className mean? A class of what? Looking at this as a complete newby to the feature so miss the context of the entire feature, its working and the lingo used.

Reply:
Having reached the end of the entire PR I think instanceName would be much better naming as it is about the instance of a spire-server installation.